command/drivetrain.py

```diff
@@ -72,31 +73,6 @@ def __init__(self, subsystem: Drivetrain):
         super().__init__(subsystem)
         self.subsystem = subsystem
 
-    def zero(self):
-        ...
-
-    def zero_success(self):
```

Comment:
What was the idea around this zero_success method, and why have we abandoned it?